app/helpers/rules_of_origin_helper.rb

```diff
@@ -18,4 +20,16 @@ def rules_of_origin_schemes_intro(country_name, schemes)
 
     safe_join schemes_intros, "\n\n"
   end
+
+  def rules_of_origin_tagged_descriptions(content)
+    content.gsub(/\{\{([A-Z]+)\}\}/) do |_match|
+      matched_tag = Regexp.last_match(1)
```

Comment:
Is this the same as match.last?

Reply:
:eyes:

Reply:
Unfortunately not - `String#gsub` returns a `String` as `_match`, not a regular expression `MatchData`  - the API docs are ambiguous but imply its MatchData but if I check in the debugger _match is definitely a String

Reply:
Think I'll go with it as is - would have much preferred to not rely on `.last_match` - anything involving globals feels ugly